cf-manifest/deployments/000-base-cf-deployment.yml

```diff
@@ -8,6 +8,8 @@ meta:
   releases:
   - name: cf
     version: latest
+  - name: nginx
+    version: latest
```

Comment:
I think adding the nginx release to do the http->https redirect is out of scope for this PR. I think for now we should restrict this PR to enabling https in gorouter.

When we come to look at the http->https redirect for this deployment, we may not necessarily want to do it in the same was as we did for the trial.